Google.Api.CommonProtos/Google.Api.CommonProtos.csproj

```diff
@@ -18,5 +17,6 @@
     <PackageReference Include="Google.Protobuf" Version="3.8.0" />
     
     <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.3" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
```

Comment:
I was expecting this to be conditional, to only be included when building on Linux?

Reply:
That was my initial expectation too, but this is basically the XML from the NuGet package page. I believe the point is that it's basically a no-op on Windows.